src/style.css

```diff
@@ -1,3 +1,56 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  background-color: white;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+  gap: 0;
+  opacity: 0;
+}
+
+.nav_list {
```

Comment:
Avoid using tag names for styling. Use class selectors instead. Consider renaming '.nav_list' to '.nav__list' to follow BEM naming conventions.